packages/reactivity/src/ref.ts

```diff
@@ -219,7 +219,9 @@ type UnwrapRefSimple<T> = T extends
   | Ref
   | RefUnwrapBailTypes[keyof RefUnwrapBailTypes]
   ? T
-  : T extends Array<any>
+  : T extends
+    | any[]
+    | readonly any[]
```

Comment:
Wouldn't `T extends ReadonlyArray<any>` be enough here instead of `T extends any[] | renadonly any[]`?

Reply:
Yes, you are right. https://www.typescriptlang.org/play?ssl=9&ssc=65&pln=9&pc=69#code/C4TwDgpgBAhgjFAvFAdgVwLYCMICcDaAugFCiSwBMSUuEMAJgPYoA2IqmOBJZ0wCyfHAA0UCj3B8qyWg2ZsoQ0eOKlJNAbAQQAHsAgp6AZygAlOk1YgAgrlwwQAHnTY8APigB+KMFxpoAFxQAGYwLEbQAPSRSB6+-mrkuNKUULr6hibmcla29k4uXB7e8YEhYRFQ0bE+fhCJ0LgAzNT8aXoGxmYW8jZ2Ds6c7l61-lBBoeFRMYhxdQ00ACytVOmdWT25-QVDuMWjZZOV1bMHQA

Reply:
Changed. 
Thanks.